app/components/find/deadline_banner_component.rb

```diff
@@ -22,6 +22,10 @@ def apply_deadline
       cycle_timetable.apply_deadline.to_fs(:govuk_date_and_time)
     end
 
+    def cycle_year_range
```

Comment:
So that these dates don't get mixed up, Instead of defining another method. could we use: 

https://github.com/DFE-Digital/publish-teacher-training/blob/ffd59fa1d2d87c770e8dd1420dacb6c33c8c47bf/app/helpers/recruitment_cycle_helper.rb#L8-L10

Reply:
I'm not sure the value of creating a new method. Would it be ok to just 
```suggestion
    def next_recruitment_cycle_period_text
      cycle_timetable.cycle_year_range
    end
```

Reply:
Had a chat about this offline, we decided it would be a good idea to start referencing the cycle timetable for cycle related dates such as this.